src/ArcanistOwnersWorkflow.php

```diff
@@ -56,6 +56,15 @@ public function requiresRepositoryAPI() {
 
   public function getArguments() {
     return array(
+      'output' => array(
+        'param' => 'format',
+        'support' => array(
+          'json',
```

Comment:
And support `text` as an option here, for completeness?

Reply:
I was [following the example of `ArcanistFeatureWorkflow` here](https://gitlab.gugud.com/imk/phabricator/arcanist/blob/64d03ff68bf2ff4ef99186472704df8aface9ef3/src/workflow/ArcanistFeatureWorkflow.php#L61-69), which only provides a `json` option (with a textual-table default) — though I agree that completeness would be better in isolation.

Do you think consistency is important here? If not I'll add the `text` option.

Reply:
If that's how other Arcanist workflows are defined, then I think that's a good measure of consistency.